src/javascript/javascript-export-scanner.ts

```diff
@@ -48,24 +48,50 @@ export class Export implements Resolvable, Feature {
 
   constructor(
       astNode: ExportNode, statementAst: babel.Statement,
-      sourceRange: SourceRange|undefined, nodePath: NodePath<babel.Node>) {
+      sourceRange: SourceRange|undefined, nodePath: NodePath<babel.Node>,
+      exportingAllFrom?: Iterable<Export>) {
     this.astNode = astNode;
     this.statementAst = statementAst;
-    for (const name of esutil.getBindingNamesFromDeclaration(astNode)) {
+    let exportedIdentifiers;
+    if (exportingAllFrom !== undefined) {
+      exportedIdentifiers = (function*() {
```

Comment:
I don't think we're using IterableX here yet, but `flatMap` would be understandable here for functional folk.

Reply:
Done, added a local definition of flatMap and called that here.